geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -1221,7 +1223,8 @@ public void addGatewaySenderId(String gatewaySenderId) {
     }
   }
 
-  public void updatePRConfigWithNewSetOfGatewaySenders(Set<String> gatewaySendersToAdd) {
+  public void updatePRConfigWithNewSetOfAsynchronousEventDispatchers(
```

Comment:
the old method name seems more appropriate here.

Reply:
The method is used indistinctly to add both a `gateway-sender` and a `async-event-queue`, that's why I changed the signature. Leaving the old method name only confuses the reader if you ask me (the name implies that it's only used to add a `gateway-sender`)... Why do you say the old method name is more appropriate?.